tests/models/mask2former/test_image_processing_mask2former.py

```diff
@@ -49,8 +51,8 @@ def __init__(
         size=None,
         do_resize=True,
         do_normalize=True,
-        image_mean=[0.5, 0.5, 0.5],
-        image_std=[0.5, 0.5, 0.5],
+        image_mean=0.5,
+        image_std=0.5,
```

Comment:
Please, revert to the original state :)

Reply:
When a revert is applied both values will become 3-element lists and will not be compatible with tests run on images with a different number of channels.

Reply:
If we want to be super precise I can write '[0.5] * num_channels'.

Reply:
Yes, that way will be better, just to be sure normalization work for this case 👍